jdaviz/cli.py

```diff
@@ -48,11 +48,17 @@ def main(filename, layout='default', instrument=None, browser='default',
         import asyncio
         asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
 
-    # Support comma-separate file list
-    filepath = ','.join([str(pathlib.Path(f).absolute()).replace('\\', '/')
-                         for f in filename.split(',')])
+    if filepaths:
+        # Convert paths to posix string; windows paths are not JSON compliant
+        file_list = [pathlib.Path(f).absolute().as_posix() for f in filepaths]
+        if layout == "imviz":
+            # Imviz multiloading should be done all at once for batch processing.
+            # Imviz convention is a single string of consecutive, comma-separated file paths
+            file_list = [','.join(file_list)]
+    else:
+        file_list = []
 
-    with open(os.path.join(CONFIGS_DIR, layout, layout + '.ipynb')) as f:
+    with open(str(JDAVIZ_DIR / "jdaviz_cli.ipynb")) as f:
```

Comment:
`open` takes `Path` object, so casting to `str` is unnecessary.

```suggestion
    with open(JDAVIZ_DIR / "jdaviz_cli.ipynb") as f:
```